docs/samples/git-setup/README.md

```diff
@@ -18,9 +21,20 @@ The content is as follows:
     * the ManagedClusterSetBinding
     * the ACM PolicyGenerator(s)
     * the source CRs
-        * The content of the `source-crs` directory should be copied over from the [cnf-features-deploy](https://github.com/openshift-kni/cnf-features-deploy/tree/master/ztp/source-crs/) repo or from the [ztp-site-generate](https://catalog.redhat.com/software/containers/openshift4/ztp-site-generate-rhel8/6154c29fd2c7f84a4d2edca1) container.
+        * The content of the `source-crs` directory should be copied over from the [cnf-features-deploy](https://github.com/openshift-kni/cnf-features-deploy/tree/master/ztp/source-crs/) repo or extracted from the [ztp-site-generate](https://catalog.redhat.com/software/containers/openshift4/ztp-site-generate-rhel8/6154c29fd2c7f84a4d2edca1) container.
         * Make sure to bring over the source CRs corresponding to the OCP release provided in the ClusterTemplate CR.
         * The ACM PGs will reference the source CRs to generate the ACM Policies that will be applied on the spoke cluster(s).
         * The source CRs included here are just samples.
 
-# Setting up ArgoCD (TODO)
\ No newline at end of file
+## Preparation of the Hub cluster
+
+Preparing the hub cluster for provisioning clusters through the O-Cloud Manager is similar to the [ZTP way](https://github.com/openshift-kni/cnf-features-deploy/tree/master/ztp/gitops-subscriptions/argocd#preparation-of-hub-cluster-for-ztp).
+
+Distinctions:
+* The `Application` CRs should point to the [clustertemplates](./clustertemplates/) and respectively the [policytemplates](./policytemplates/) directories.
+* The following should be added to the `AppProject` for `clustertemplates`, under `spec.clusterResourceWhitelist`:
```

Comment:
Maybe add this link https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/gitops-subscriptions/argocd/deployment/app-project.yaml to `AppProject`?